### ISSUE TITLE
json: cannot unmarshal object into Go value of type []*xgboost.xgboostJSON

### ISSUE THREAD

**Issue by szmcdull** [Elvenson/xgboost-go#16]
How to load DMLC XGBoost model (https://xgboost.readthedocs.io/en/stable/tutorials/saving_model.html)

![image](https://github.com/Elvenson/xgboost-go/assets/2098590/c82a224b-1b16-443a-801f-c91520fb48e4)

**Comment by szmcdull** [comment]
Sorry, i then read the doc. I should use dump_model instead of save_model

**Comment by sedletsky-f5** [comment]
> Sorry, i then read the doc. I should use dump_model instead of save_model

right, it is a typical mistake (happened to me initially) even if there is a doc for this. duality of dump/save for Go/python is the reason...
